src/zenml/integrations/gcp/image_builders/gcp_image_builder.py

```diff
@@ -177,9 +177,12 @@ def _configure_cloud_build(
         # Convert the docker_build_options dictionary to a list of strings
         docker_build_args = []
         for key, value in build_options.items():
+            option = f"--{key}"
             if isinstance(value, list):
                 for val in value:
-                    docker_build_args.extend([key, val])
+                    docker_build_args.extend([option, val])
+            elif value is None or isinstance(value, bool):
+                docker_build_args.append(option)
```

Comment:
Now passing `rm=False` as an option would be translated to `--rm` right?

(I don't have any idea how to solve this in a nice way btw, seems impossible)